.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
     publish:
         docker:
-            - image: circleci/node:12.9.1-browsers
+            - image: circleci/node:14.9-browsers
```

Comment:
This should also be fine. I've had to update it in a couple other repos already to use some newer features.